### PR TITLE
Portfolio details popup window

### PR DESCRIPTION
**Pull request by MuhmmdUsama**
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- popups window for both mobile and desktop screen sizes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -105,7 +113,9 @@ <h2>
               <li>JavaScript</li>
               <li>html</li>
             </ul>
-            <button type="button" class="See-Project">See Project</button>
+            <button type="button" class="See-Project pop-btn ">
+              See Project
+            </button>
```
Comment (CollinsTatang) [inline index.html:112]: - [x] Please fix the `button` to display the project details when a user clicks on it.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x] Please do your possible best as a team to follow the project [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md).

- [x] Each team member should make multiple commits to the milestone changes since is a group project.

- [x] Please design your project according to one of the [Figmal Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MuhmmdUsama) [reply index.html:112]: @CollinsTatang 
I don't get your point, the button works fine and the pop-up window work fine gives me more explanation plz

Comment (CollinsTatang) [reply index.html:112]: Hi @MuhmmdUsama 
I run your project and the button where not working.
### **Take a Look**

https://user-images.githubusercontent.com/55596970/162109663-1d18e23f-00c2-4f38-8114-aaff26cdd421.mp4

Comment (MuhmmdUsama) [reply index.html:112]: ok i got it, thanks

Comment (Whoistolu) [reply index.html:112]: - [x] [ optional] Hello @MuhmmdUsama You have done a great implementation. 
On the mobile popup, kindly remove the excess space beneath to make your project complete. I made this optional as that is not enough not to approve it but please ensure you fix this.
### Your Design
Screenshot from 2022-04-07 10-47-53

**Review by Whoistolu — APPROVED** [review]
### Hi @MuhmmdUsama

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [x]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MuhmmdUsama) [reply index.html:112]: @Whoistolu 
thank you so much I will fix it

**Pull request merged**